src/devices/cpu/dsp56156/dsp56ops.hxx

```diff
@@ -216,24 +227,779 @@ static void dsp56156_process_rep(dsp56156_core* cpustate, size_t repSize);
 /********************/
 /* Helper Functions */
 /********************/
-static uint16_t Dsp56156OpMask(uint16_t op, uint16_t mask);
+static uint16_t dsp56156_op_mask(uint16_t op, uint16_t mask);
+
+static uint64_t dsp56156_limit_long_to_long(dsp56156_core* cpustate, uint64_t val);
 
 /* These arguments are written source->destination to fall in line with the processor's paradigm. */
-static void SetDestinationValue(typed_pointer source, typed_pointer dest);
+static void dsp56156_set_destination_value(dsp56156_core* cpustate, typed_pointer source, typed_pointer dest);
+
+static void dsp56156_set_data_memory_value(dsp56156_core* cpustate, typed_pointer source, uint32_t destinationAddr);
+static void dsp56156_set_program_memory_value(dsp56156_core* cpustate, typed_pointer source, uint32_t destinationAddr);
+
+
+
+/***************************************************************************
+    IMPLEMENTATION
+***************************************************************************/
+
+size_t get_op_size(uint16_t op, uint16_t op2)
```

Comment:
This has a generic name and is externally visible – can you make it `static` or `inline` to avoid potential duplicate symbols?

Reply:
Made static as well as prefixed with `dsp56156_` to be more consistent with other static functions in the file.